### ISSUE TITLE
HedgeEdit - Spline Previewing

### ISSUE THREAD

**Issue by Radfordhound** [Radfordhound/HedgeLib#23]
It'd be nice to actually be able to see splines.
Shouldn't be hard in the DirectX version since Direct3D actually has built-in support for drawing lines, unlike modern OpenGL for some reason.

**Comment by KTE2399** [comment]
Modern OpenGL has built in support for drawing lines that grid is drawn using GL_LINES

![image](https://user-images.githubusercontent.com/16513942/45614064-0d86ee80-ba8a-11e8-8342-5d7b855460c8.png)

**Comment by thesupersonic16** [comment]
I have only used OpenGL 1.1, And I know that has Line Rendering.

**Comment by Radfordhound** [comment]
Yeah, I knew GL 1.1 had it. Somehow I didn't realize you could still specify PrimitiveType.Lines with OpenTK when using modern GL. Ah well...
